### PR TITLE
rename TabNine to tabnine

### PR DESCRIPTION
**Pull request by dimacodota**

### CODE REVIEW

File: Main.sublime-menu [hunk 1]
```diff
@@ -6,7 +6,7 @@
                 "id": "package-settings",
                 "children": [
                     {
-                        "caption": "TabNine",
+                        "caption": "tabnine",
```
Comment (boaz-codota) [inline Main.sublime-menu:9]: Maybe add the icon thingy here as well?

File: lib/logger.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 def info(msg):
-    print("[TabNine] {} | {}".format(_time(), msg))
+    print("[tabtine] {} | {}".format(_time(), msg))
```
Comment (boaz-codota) [inline lib/logger.py:8]: Oopsi

File: lib/tab_nine_process.py [hunk 3]
```diff
@@ -129,7 +129,7 @@ def request(self, req):
             result = loads(result)
             return result
         except (IOError, OSError, UnicodeDecodeError, ValueError) as e:
-            print("Exception while interacting with TabNine subprocess:", e)
+            print("Exception while interacting with tabnine subprocess:", e)
```
Comment (boaz-codota) [inline lib/tab_nine_process.py:132]: Should be `Tabnine` here

**Review by boaz-codota — CHANGES_REQUESTED** [review]

File: Main.sublime-menu (hunk 1 above)
Comment (dimacodota) [reply Main.sublime-menu:9]: cool 👍 
image
image

**Review by boaz-codota — APPROVED** [review]

**Pull request merged**